modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/resources/com/liferay/portal/workflow/kaleo/dependencies/single-approver-failed-scripted-assignment-workflow-definition.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0"?>
```

Comment:
@pedro-oliveira446 currently we are using .json instead of .xml, see this example below:

https://github.com/liferay/liferay-portal/blob/92f257241e2e88c13859d421ebf74c2a5ac1bdba/modules/apps/portal-security/portal-security-script-management-test/src/testIntegration/resources/com/liferay/portal/security/script/management/web/internal/portlet/action/test/dependencies/workflow-definition-1.json

Reply:
tip: you can leverage workflow admin API performing GET operation to retrieve your workflow definition content as json.

Reply:
Done!